lib/rules/no-var.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @fileoverview Rule to check for the usage of var.
+ * @author Jamund Ferguson
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = function(context) {
+
+    return {
+        "VariableDeclaration": function (node) {
+            if (node.kind === "var") {
+                context.report(node, "Please use \"let\" instead of \"var\".");
```

Comment:
We usually phrase these messages in the form "Expected x" or "Unexpected y", so like "Unexpected var; use let or const instead."